DataFormats/Reconstruction/include/ReconstructionDataFormats/Track.h

```diff
@@ -92,6 +92,7 @@ constexpr float kCY2max = 100 * 100, // SigmaY<=100cm
   kCSnp2max = 1 * 1,                 // SigmaSin<=1
   kCTgl2max = 1 * 1,                 // SigmaTan<=1
   kC1Pt2max = 100 * 100,             // Sigma1/Pt<=100 1/GeV
+  kMostProbablePt = 0.35,            // Most Probable Pt (GeV), for running with Bz=0
```

Comment:
Isn't it too low? For the TPC I asked David to use 0.6 GeV.

Reply:
@shahor02  I took this value from AliRoot. It may correspond to the most probable pt (the position of the maximum of the spectrum).
But you are right. We should better use the *mean* pt, which is higher.

I will change it to 0.6 GeV.  And I will also add a setter, to be able to set it to something even higher. Which could be useful for cosmic runs.